supfile.go

```diff
@@ -13,21 +13,66 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// EnvVar represents an environment variable
+type EnvVar struct {
+	Key   string
+	Value string
+}
```

Comment:
Mostly it's about hiding the `interface{}` which would be annoying to work with later on. Rather do a bit of work upfront and then having an actual sane type to work with.